src/SFA.DAS.AssessorService.Web.UnitTests/Validators/Standard/OptOutStandardVersionViewModelValidatorTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Microsoft.AspNetCore.Http;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AssessorService.Api.Types.Models.Standards;
+using SFA.DAS.AssessorService.Application.Api.Client.Clients;
+using SFA.DAS.AssessorService.Web.Validators.Standard;
+using SFA.DAS.AssessorService.Web.ViewModels.Standard;
+using SFA.DAS.Testing.AutoFixture;
+using System.Security.Claims;
+
+namespace SFA.DAS.AssessorService.Web.UnitTests.Validators.Standard
+{
+    public class OptOutStandardVersionViewModelValidatorTests
+    {
+        [Test, MoqAutoData]
+        public void MultipleApprovedStandardVersions_Passes()
+        {
+            var viewModel = new OptOutStandardVersionViewModel() { Version = "1.1", StandardReference = "ST999" };
+            var standardVersionOne = new StandardVersion() { Version = "1.1" };
+            var standardVersionTwo = new StandardVersion() { Version = "1.2" };
+
+            var httpContextAccessor = GetMockHttpContextAccessor();
+            var apiClient = GetMockApiClient(standardVersionOne, standardVersionTwo);
+
+            var sut = new OptOutStandardVersionViewModelValidator(httpContextAccessor.Object, apiClient.Object);
+
+            var result = sut.Validate(viewModel);
+
+            Assert.That(result.IsValid, Is.True);
+        }
+
+        [Test, RecursiveMoqAutoData]
+        public void OnlyOneApprovedStandardVersion_Fails_WithErrorMessage(OptOutStandardVersionViewModel viewModel,
+                                                                          StandardVersion version)
+        {
+            var httpContextAccessor = GetMockHttpContextAccessor();
+            var apiClient = GetMockApiClient(version);
+
+            var sut = new OptOutStandardVersionViewModelValidator(httpContextAccessor.Object, apiClient.Object);
+
+            var result = sut.Validate(viewModel);
+
+            Assert.Multiple(() =>
+            {
+                Assert.That(result.IsValid, Is.False);
+                Assert.That(result.Errors.Exists(e => e.ErrorMessage == OptOutStandardVersionViewModelValidator.OneApprovedVersionRequiredMessage));
+            });
+        }
+
+        [Test]
+        public void EmptyStandardReference_FailsValidation()
```

Comment:
I don't think an empty standard reference should fail validation it would be an exception, so the exception should be throw from the validator

Reply:
Resolving this since we've agreed to both throw in the controller and validate in the validator